website_blog_mail_server/models/website.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    website_blog_mail_server_id = fields.Many2one(
```

Comment:
```suggestion
    blog_mail_server_id = fields.Many2one(
```

I prefer field names to be concise.